wp-content/plugins/wporg-learn/views/block-workshop-details.php

```diff
@@ -17,6 +23,14 @@
 					<span><?php echo esc_html( $value ); ?></span>
 				</li>
 			<?php endforeach; ?>
+
+			<?php if ( $has_transcript ) : ?>
+				<li>
+					<a class="components-button is-secondary is-small" href="#transcript">
```

Comment:
This button looks kind of out-of-place in the Workshop Details list. What if it matched the other detail rows more, something like this?

![transcript](https://user-images.githubusercontent.com/916023/119061560-805a6d00-b989-11eb-886b-fc34a4db5261.jpg)

Reply:
I like that, done in 708cf87ca32cb6d665293d3da990984a7d9fd870